src/main/java/it/gov/pagopa/payhub/activities/service/ingestionflow/organization/OrganizationProcessingService.java

```diff
@@ -0,0 +1,106 @@
+import it.gov.pagopa.payhub.activities.connector.organization.BrokerService;
+import it.gov.pagopa.payhub.activities.connector.organization.OrganizationService;
+import it.gov.pagopa.payhub.activities.dto.ingestion.organization.OrganizationErrorDTO;
+import it.gov.pagopa.payhub.activities.dto.ingestion.organization.OrganizationIngestionFlowFileDTO;
+import it.gov.pagopa.payhub.activities.dto.ingestion.organization.OrganizationIngestionFlowFileResult;
+import it.gov.pagopa.payhub.activities.mapper.ingestionflow.organization.OrganizationMapper;
+import it.gov.pagopa.payhub.activities.service.ingestionflow.IngestionFlowProcessingService;
+import it.gov.pagopa.pu.organization.dto.generated.Broker;
+import it.gov.pagopa.pu.organization.dto.generated.Organization;
+import it.gov.pagopa.pu.processexecutions.dto.generated.IngestionFlowFile;
+import java.nio.file.Path;
+import java.util.ArrayList;
+import java.util.Iterator;
+import java.util.List;
+import java.util.Optional;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.context.annotation.Lazy;
+import org.springframework.stereotype.Service;
+
+@Service
+@Lazy
+@Slf4j
+public class OrganizationProcessingService extends IngestionFlowProcessingService<OrganizationIngestionFlowFileDTO, OrganizationIngestionFlowFileResult, OrganizationErrorDTO> {
+
+    private final OrganizationMapper organizationMapper;
+    private final OrganizationService organizationService;
+
+
+    public OrganizationProcessingService(
+            OrganizationMapper organizationMapper,
+            OrganizationErrorsArchiverService organizationErrorsArchiverService,
+        OrganizationService organizationService) {
+        super(organizationErrorsArchiverService);
+        this.organizationMapper = organizationMapper;
+        this.organizationService = organizationService;
+    }
+
+
+    public OrganizationIngestionFlowFileResult processOrganization(
+            Iterator<OrganizationIngestionFlowFileDTO> iterator,
+            List<CsvException> readerException,
+            IngestionFlowFile ingestionFlowFile, Path workingDirectory) {
+        List<OrganizationErrorDTO> errorList = new ArrayList<>();
+        OrganizationIngestionFlowFileResult ingestionFlowFileResult = new OrganizationIngestionFlowFileResult();
+        ingestionFlowFileResult.setOrganizationIpaCodeList(new ArrayList<>());
+
+        Optional<Organization> organizationBroker = organizationService.getOrganizationById(ingestionFlowFile.getOrganizationId());
+        Long brokerId = organizationBroker.map(Organization::getBrokerId).orElse(null);
+        if (brokerId == null) {
+            log.error("Broker for organization id {} not found", ingestionFlowFile.getOrganizationId());
+            OrganizationErrorDTO error = new OrganizationErrorDTO(
+                ingestionFlowFile.getFileName(), organizationBroker.get().getIpaCode(),
+                null, "BROKER_NOT_FOUND", "Broker not found");
+            errorList.add(error);
```

Comment:
this should return a
OrganizationIngestionFlowFileResult
having errorDescription `Broker not found`